docs/openapi/specification.yml

```diff
@@ -1,35 +1,296 @@
 openapi: '3.0.0'
 info:
-  title: GOV.UK Search API
+  title: GOV.UK Search API v2
   version: 2.0.0-alpha
 
 paths:
   /search:
```

Comment:
Should this be /search.json (I notice it's inconsistent with the auto complete one)

Reply:
Actually it can just be `/search` (the `.json` is just a Railsism after all and the request content type is what matters - not to mention even that is ignored by Search API v2 which will always return JSON regardless of what you ask for 😂 )

Reply:
Great - I saw you amended autocomplete instead. Looks good.